Examples/javascript/array/runme.js

```diff
@@ -0,0 +1,7 @@
+var example = require("example");
+
+var arr = [1, 2, 3, 4, 5];
+
+console.log(example.sum1(arr, arr.length));
+console.log(example.sum2(arr));
+console.log(example.data3)
```

Comment:
This doesn't fail if the results are incorrect, it just writes it to the console and without any context there's basically no chance anyone will notice if they're wrong.